### PR TITLE
doc: fixing headings, mention in README.md, adding TailwindCSS steps

### PR DESCRIPTION
**Pull request by spekulatius**
Hello @Kocal 

as promised some more tidy up on the documentation site. This includes:

- Fixing headings to have a consistent writing
- Mention the documentation site in the project README.md
- Adding configuration steps for TailwindCSS

If you think anything needs to be tweaked, just let me know and I'll do it before merging :)

Cheers,
Peter

### CODE REVIEW

File: docs/development/TailwindCSS.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            return contentWithoutStyleBlocks.match(/[A-Za-z0-9-_/:]*[A-Za-z0-9-_/]+/g) || [];
+        },
+        },
+    ],
+
+    whitelist: [],
+    whitelistPatterns: [
+        /-(leave|enter|appear)(|-(to|from|active))$/, // transitions
+        /data-v-.*/, // scoped css
+    ],
```
Comment (Kocal) [inline docs/development/TailwindCSS.md:55]: To indent

File: docs/development/TailwindCSS.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    });
+
+    module.exports = {
+    plugins: [tailwindcss(), ...(process.env.NODE_ENV === 'production' ? [purgecss] : [])],
```
Comment (Kocal) [inline docs/development/TailwindCSS.md:59]: To indent

**Comment by spekulatius** [comment]
> Thank you @spekulatius! It's look good to me, with few minor comments.

Ah, thanks for checking. Sorry, I've only copied it over and didn't check the indentation. It should be fixed now.

**Comment by Kocal** [comment]
Thank you!

**Comment by spekulatius** [comment]
No problem!

**Pull request merged**